src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -266,6 +266,18 @@ public abstract partial class TransportContext
         protected TransportContext() { }
         public abstract System.Security.Authentication.ExtendedProtection.ChannelBinding GetChannelBinding(System.Security.Authentication.ExtendedProtection.ChannelBindingKind kind);
     }
+
+    public static class HttpVersion
```

Comment:
Even though it really *should* be a `static` class, `HttpVersion` is not `static` in .NET Standard (it's not sealed and has a public ctor).

https://github.com/dotnet/standard/blob/4f2b0f018eb22a28fb2ebbccdcaa4f8efbc45183/netstandard/ref/System.cs#L5262

Can we change it to be `static` in .NET Standard, or do we have to keep it as-is in the off chance that someone out there is newing up an instance or is subclassing it?